docs/content/_navigation.json

```diff
@@ -477,50 +477,7 @@
   {
     "title": "Guides",
     "icon": "Search",
-    "path": "/guides",
```

Comment:
removing all child items feels odd that i can no longer find these individual guides in the left nav or can't see the item show up on the left when im inside a particular guide.

in order to organize the nav better, could we instead introduce extra layers:
```
Guides
    - General
    - Best practices
    - Experimental features
```
like this: <img width="200" alt="image" src="https://user-images.githubusercontent.com/4531914/183216403-1e0cc481-5fab-494c-99db-3880e1d94b41.png">

Reply:
I had this thought too - do we want to add additional pages for this? Sandy mentioned it was odd to have the sidenav link to subheadings in a page.